lib/rspecq/worker.rb

```diff
@@ -156,20 +162,24 @@ def reset_rspec_state!
     # Their errors will be reported in the normal flow, when they're picked up
     # as jobs by a worker.
     def files_to_example_ids(files)
-      # TODO: do this programatically
-      cmd = "DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')}"
+      cmd = "DISABLE_SPRING=1 bundle exec rspec --dry-run --format json #{files.join(' ')} 2>&1"
       out = `#{cmd}`
 
       if !$?.success?
-        # TODO: emit warning to Sentry
-        puts "WARNING: Error splitting slow files; falling back to regular scheduling:"
-
-        begin
-          pp JSON.parse(out)
+        rspec_output = begin
```

Comment:
Indentation could possibly be improved for the `begin/rescue` block.

Reply:
Fixed